### DISCUSSION TITLE
2022.1.29 build failed

### DISCUSSION THREAD

**Discussion by Obegg** [zhongfly/mpv-build#11] [General]
New build failed

**Comment by zhongfly** [comment]
Because of ffmpeg
https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/hwcontext_qsv.c#L137
```
src/libavutil/hwcontext_qsv.c: In function 'ff_qsv_get_surface_base_handle':
src/libavutil/hwcontext_qsv.c:137:39: error: 'mfxHDLPair' has no member named 'secode'; did you mean 'second'?
  137 |         base_handle[1] = handle_pair->secode;
      |                                       ^~~~~~
      |                                       second
make: *** [src/ffbuild/common.mak:78: libavutil/hwcontext_qsv.o] Error 1
make: *** Waiting for unfinished jobs....
```

**Reply by zhongfly** [reply]
we should wait ffmpeg to fix it

**Reply by zhongfly** [reply]
BtbN's ffmpeg build also failed: https://github.com/BtbN/FFmpeg-Builds/actions/runs/1765724701

**Reply by zhongfly** [reply]
have fixed https://github.com/FFmpeg/FFmpeg/commit/2f323b19789dac87ec57b9fec663e049e0dc4164

**Reply by Obegg** [reply]
Can you manually build mpv (instead of waiting for tomorrow) so I will have the final version of auto-update and daily updates, please?

**Reply by zhongfly** [reply]
> Can you manually build mpv (instead of waiting for tomorrow) so I will have the final version of auto-update and daily updates, please?

I want my releases to be regular and timed and I don't want to disrupt my release cycle so I don't manually release updates right now. 
If you want today  daily build, please wait https://github.com/zhongfly/mpv-winbuild/actions/runs/1766185170 ,but I will not create release for this.
